operations/helm/scripts/build.sh

```diff
@@ -83,7 +83,7 @@ function generate_manifests() {
   helm template "${ARGS[@]}" 1>/dev/null
   cp -r "${INTERMEDIATE_OUTPUT_DIR}" "${OUTPUT_DIR}"
   rm "${OUTPUT_DIR}/${CHART_NAME}/templates/values-for-rego-tests.yaml"
-  find "${OUTPUT_DIR}/${CHART_NAME}/templates" -type f -print0 | xargs -0 "${SED}" -E -i -- "/^\s+(checksum\/(alertmanager-fallback-)?config|(helm.sh\/)?chart|app.kubernetes.io\/version|image: \"grafana\/(mimir|mimir-continuous-test|enterprise-metrics)):/d"
+  find "${OUTPUT_DIR}/${CHART_NAME}/templates" -type f -print0 | xargs -0 "${SED}" -E -i -- "/^\s+(checksum\/(alertmanager-fallback-)?config|(helm.sh\/)?chart|app.kubernetes.io\/version|image: \"?grafana\/(mimir|mimir-continuous-test|enterprise-metrics)):/d"
```

Comment:
This one fixes the build script, so it searched both quoted and not-quoted reference to the default image:

```
image: "grafana/mimir:
````

and

```
image: grafana/mimir:
```

(note the quote `"`)

That is, we don't want the generated manifests to have the default image, so we didn't have to re-generate them everytime the default image changes (e.g. on every weekly release).